diff -Naur steam.old/lib/udev/rules.d/60-steam-input.rules steam/lib/udev/rules.d/60-steam-input.rules
--- steam.old/lib/udev/rules.d/60-steam-input.rules	2020-02-09 21:13:29.661326479 +0100
+++ steam/lib/udev/rules.d/60-steam-input.rules	2020-02-09 21:13:54.782774756 +0100
@@ -49,6 +49,9 @@
 # EMiO Elite Controller for PS4
 KERNEL=="hidraw*", ATTRS{idVendor}=="0c12", ATTRS{idProduct}=="1cf6", MODE="0660", TAG+="uaccess"
 
+# ZeroPlus P4 (hitbox)
+KERNEL=="hidraw*", ATTRS{idVendor}=="0c12", ATTRS{idProduct}=="0ef6", MODE="0660", TAG+="uaccess"
+
 # HORI RAP4
 KERNEL=="hidraw*", ATTRS{idVendor}=="0f0d", ATTRS{idProduct}=="008a", MODE="0660", TAG+="uaccess"
 
