diff -Naur steam.old/lib/udev/rules.d/99-steam-controller-perms.rules steam/lib/udev/rules.d/99-steam-controller-perms.rules
--- steam.old/lib/udev/rules.d/99-steam-controller-perms.rules	2016-11-28 14:49:17.909244287 +0100
+++ steam/lib/udev/rules.d/99-steam-controller-perms.rules	2016-11-28 14:50:15.374202514 +0100
@@ -23,3 +23,7 @@
 
 # DualShock 4 Slim over bluetooth hidraw
 KERNEL=="hidraw*", KERNELS=="*054C:09CC*", MODE="0666"
+
+# This rule is necessary for gamepad emulation; make sure each user requiring
+# it is part of the input group.
+KERNEL=="uinput", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"
