diff -Naur steam.old/lib/udev/rules.d/60-steam-input.rules steam/lib/udev/rules.d/60-steam-input.rules
--- steam.old/lib/udev/rules.d/60-steam-input.rules	2018-10-11 09:43:36.771531454 +0200
+++ steam/lib/udev/rules.d/60-steam-input.rules	2018-10-11 09:46:09.264176676 +0200
@@ -62,3 +62,7 @@
 
 # STRIKEPAD PS4 Grip Add-on
 KERNEL=="hidraw*", ATTRS{idVendor}=="054c", ATTRS{idProduct}=="05c5", MODE="0666"
+
+# This rule is necessary for gamepad emulation; make sure each user requiring
+# it is part of the input group.
+KERNEL=="uinput", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"
