diff -Naur steam.old/steam steam/steam
--- steam.old/steam	2015-05-25 18:22:27.579706560 +0200
+++ steam/steam	2015-05-25 18:21:09.342722219 +0200
@@ -195,3 +195,7 @@
 
 # Workaround for bug 3570: https://github.com/ValveSoftware/steam-for-linux/issues/3570
 rm -f /tmp/steam_chrome_shmem
+
+# Workaround for bug 3273: https://github.com/ValveSoftware/steam-for-linux/issues/3273
+rm -f $LAUNCHSTEAMPLATFORM/steam-runtime/i386/usr/lib/i386-linux-gnu/libstdc++.so.6*
+rm -f $LAUNCHSTEAMPLATFORM/steam-runtime/amd64/usr/lib/x86_64-linux-gnu/libstdc++.so.6*
